src/Behat/Behat/Transformation/Context/Attribute/TrasnformationAttributeReader.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Behat.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Behat\Transformation\Context\Attribute;
+
+use Behat\Behat\Context\Annotation\DocBlockHelper;
+use Behat\Behat\Context\Attribute\AttributeReader;
+use Behat\Behat\Transformation\Context\Annotation\TransformationAnnotationReader;
+use Behat\Transformation as Attribute;
+use ReflectionMethod;
+
+/**
+ * Reads transformation Attributes from the context class.
+ *
+ * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ */
+final class TrasnformationAttributeReader extends TransformationAnnotationReader implements AttributeReader
+{
+    /**
+     * @var DocBlockHelper
+     */
+    private $docBlockHelper;
+
+    /**
+     * Initializes reader.
+     *
+     * @param DocBlockHelper $docBlockHelper
+     */
+    public function __construct(DocBlockHelper $docBlockHelper)
+    {
+        $this->docBlockHelper = $docBlockHelper;
+    }
+
+    /**
+     * @{inheritdoc}
+     */
+    public function readCallees(string $contextClass, ReflectionMethod $method)
+    {
+        $attributes = $method->getAttributes(Attribute\Transformation::class, \ReflectionAttribute::IS_INSTANCEOF);
+
+        $callees = [];
+        foreach ($attributes as $attribute) {
+            $docLine = '@Transform';
+            $pattern = $attribute->newInstance()->pattern;;
```

Comment:
the `Transformation` interface does not guarantee the existence of such property. If your code supports only the `Transform` class, why not using that class directly when loading attributes, instead of adding the internal interface that works only for that class ?

Reply:
I followed the structure used for other attributes which they all define a common interface for each class. But you are right that if there is only one attribute of this class the interface is not really needed, removed